src/components/common/ColonyHome/ColonyDomainSelector/ColonyDomainSelector.tsx

```diff
@@ -2,7 +2,7 @@ import React, { ReactNode, useCallback } from 'react';
 // import { ColonyVersion, ROOT_DOMAIN_ID, Extension } from '@colony/colony-js';
```

Comment:
The `ROOT_DOMAIN_ID` constant in this file can be replaced by `Id.RootDomain` from `colony-js`